cmd/distribute/distribute.go

```diff
@@ -101,8 +135,31 @@ func distributeReward() error {
 				return err
 			}
 		}
+
+		// Append bucket information to the list. For using the api to distribute the reward to the bucket.
+		for i := range toAccountAddrList {
+			allToBucketAddressList =
+				append(allToBucketAddressList,
+					toAccountAddrList[i])
+			allToBucketIDList =
+				append(allToBucketIDList,
+					toBucketIDList[i])
+			allToBucketAmountList =
+				append(allToBucketAmountList,
+					toBucketAmountList[i])
+			allToBucketDelegateNameList =
```

Comment:
repeat append?

Reply:
Several voters vote for a delegate. To process each voter, the contract is processed in the lower loop of the delegate loop. In other methods, the corresponding relationship is gone. So use multiple lists of the same length to describe which delegate the voter is, so there will be delegates with the same name.